docs/build/build-hrmp-channels.md

```diff
@@ -59,27 +59,27 @@ identity.
 Your proposal should contain the following sections to be considered complete for the community to
 review and ultimately vote:
 
-1.  A request on what the proposal aims to do (opening an HRMP channel with Statemint);
+1.  A request on what the proposal aims to do (opening an HRMP channel with Asset Hub);
 2.  The use cases this channel will support for your chain;
 3.  Technical details of the proposal, including proposal parameters and technical details of this
     call (On Kusama, most proposals were designed as a batchAll calls) :
 
-    - A force transfer from Polkadot treasury to Statemint as deposit to accept and open an HRMP
+    - A force transfer from Polkadot treasury to Asset Hub as deposit to accept and open an HRMP
       channel with your chain;
-    - Send XCM message to Statemint to execute a transaction with superuser (root) permission.
+    - Send XCM message to Asset Hub to execute a transaction with superuser (root) permission.
```

Comment:
Renaming fine but instructions way out of date

Reply:
@CrackTheCode016 can you update this section or point me out to some material? @joepetrowski any suggested changes here? Thanks in advance!

Reply:
@CrackTheCode016 please use the latest referenda https://polkadot.polkassembly.io/referendum/123 to refine this. Also, now this has to go through OpenGov (possibly only through Root or Whitelist tracks?)

Reply:
This can be done through a different PR. Here is the issue to track https://github.com/w3f/polkadot-wiki/issues/4970